### ISSUE TITLE
The emoticons are not always displayed in the PDF exported version

### ISSUE THREAD

**Issue by TechWatching** [slidevjs/slidev#792]
**Describe the bug**
The emoticons are not always displayed in the PDF exported version.

**To Reproduce**
1. Go to https://ei8it213-slides.vercel.app/2
2. Click on donwload pdf button
3. Scroll down to slide 2
4. See the emoticons are not displayed like on the SPA.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Microsoft Edge
 - Slidev version:

**Comment by twitwi** [comment]
It is an issue with browsers (chromium) printing to pdf.
As a workaround, one can use the corresponding "icons", e.g. `<twemoji-bullseye/>`.

I'm have worked on an addons that replaces emoji (characters) by icons.
It requires https://github.com/slidevjs/slidev/pull/787 to work as a plugin.
But, you can use it right now with the current version of slidev, by copying https://github.com/twitwi/slidev-addon-remoji/blob/main/setup/preparser.ts into your project (into a setup/ folder).

**Comment by TechWatching** [comment]
Thanks for the quick answer. I will use your workaround, it seems the easiest.

**Comment by naquiroz** [comment]
This fix is broken.

**Comment by naquiroz** [comment]
@twitwi Is there a way to bring the emojis back? I'v tried adding the font into Vercel but had no luck

**Comment by twitwi** [comment]
@naquiroz I just tried from a new slidev project and both using directly the components like `<twemoji-bullseye/>` and 🎯 with the addon worked for pdf export.
Are you getting errors somewhere? 

Maybe some emojis are not existing or not with the expected name.
This is the case with the personal-computer emoji used in the default presentation, I just realized there is actually an issue opened https://github.com/twitwi/slidev-addon-remoji/issues/1 but I receive too much github notification and missed it.
